src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -0,0 +1,708 @@
+import React from 'react';
```

Comment:
This is where the bulk of the content display logic lives. There are 25 different enrollment status types. Depending in what the user's status is, we have to display different content. If you are interested in learning more about this, you can start by reviewing [this doc](https://github.com/department-of-veterans-affairs/vets.gov-team/files/2912428/HCA.LOA3.Warning.Copy.Guide.docx) and then looking at [this spreadsheet](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/HealthApplication/HCA%20Duplicate%20Submissions%20Updates/Discovery%20and%20Research/Enrollment%20Status%20Discovery/HC%20Enrollment%20Categories%20and%20Display%20Text.xlsx) that breaks down which text to show the user depending on their enrollment status.